app/services/user.js

```diff
@@ -0,0 +1,16 @@
+const { user: User } = require('../models'),
+  { checkPassword } = require('../helpers'),
+  logger = require('../logger'),
+  error = require('../errors'),
+  { generateToken } = require('./jwt-encryption');
+
+exports.checkLogin = credentials => {
+  const { username, password } = credentials;
+  return User.getByUsername(username).then(data => {
+    if (data && checkPassword(password, data.password)) {
+      return generateToken(data);
+    }
+    logger.error(error);
```

Comment:
what are you logging here?

Reply:
What happens if any error is thrown while executing this function (`checkLogin `)?